.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "python.defaultInterpreterPath": "venv/bin/python",
```

Comment:
Do we need this file here? I usually try to avoid having `.vscode` config files in repos, as many of these things can come down to personal taste. For example, I don't use `venv` for Python myself - I use `conda` (the most common toolchain in my field).

Happy to be won over though.

Reply:
That's a good point. What do you think about converting that to something like, `settings.template.jsonc`, with some useful commentary on the settings for those that want to opt-in?

`jsonc` (json with comments) is a supported format in vscode that allows adding commentary to json files.

Reply:
Sure, feels slightly overkill to be but wouldn't do any harm, so happy to have that if you'd like it 😀 

> `jsonc` (json with comments) is a supported format in vscode that allows adding commentary to json files.

Awesome - never heard of this file format before! I always just use YAML instead if I can 😅